icaruscode/CRT/CRTUtils/CRTT0MatchAlg.h

```diff
@@ -81,6 +103,37 @@ namespace icarus{
 
   };
 
+  struct  matchCand_PCA {
+    sbn::crt::CRTHit thishit;
+    double t0 = DBL_MIN;
+    double dca = DBL_MIN;
+    double extrapLen = DBL_MIN;
+    int best_DCA_pos = -1;//-1=no match; 0=startdir is best; 1=enddir is best; 2=both start+end are equally good;
+    bool simple_cathodecrosser = false;
+    int driftdir = -5;
```

Comment:
What does -5 represent? I would think this value is +/- 1. If it's just a default value, maybe NULL would be better so that you can catch cases where this isn't set correctly, where -5 would just give you a large shift value (I think).

Reply:
would NULL get translated to 0 if the int is used in multiplication, though, or would it create a NaN?